Client/mods/deathmatch/logic/luadefs/CLuaBlipDefs.cpp

```diff
@@ -60,14 +60,14 @@ int CLuaBlipDefs::CreateBlip(lua_State* luaVM)
 {
     CVector          vecPosition;
     unsigned char    ucIcon = 0;
-    unsigned char    ucSize = 2;
+    int              iSize = 2;
```

Comment:
has the size param changed from an `unsigned char` to `int` to satisfy `Clamp`?

Reply:
That, but also to make sure if you passed a negative value as the argument it would take that negative value and clamp it to 0 instead of some random value between the min and max depending on the integer overflow amount.